### DISCUSSION TITLE
Phoenixd integration

### DISCUSSION THREAD

**Discussion by EdoardoSnodato** [ZeusLN/zeus#2175] [General]
Future release, is possible connect Zeus to phoenixd ? 

https://phoenix.acinq.co/server

**Comment by Darth-Coin** [comment]
It doesn't make sense to integrate, Phoenixd is just a server like Phoenix mobile wallet, but can be run on a PC. Nothing else. So just using Phoenix wallet is the same.

**Reply by mandelmonkey** [reply]
But you can't connect Phoenix wallet to your own Phoenixd right?

**Reply by Darth-Coin** [reply]
No. I think you do not understand the architecture of Phoenix mobile and phoenixd server.
Both are in fact clients for ACINQ node infrastructure. Both are opening channels ONLY with ACINQ, in their particular manner. All the liquidity is managed by ACINQ, you are just renting it from them.
phoenixd server is the same as Phoenix mobile, is just a desktop server. Is like running Phoenix mobile on a PC. That's it.
- https://phoenix.acinq.co/server
- https://github.com/mrboatsman/awesome-phoenixd

What you can do if you really want to use a phoenixd server with Zeus, is to install phoenixd on your PC, open a channel (with ACINQ automatically) and on top of that install AlbyHub and use the phoenixd as backend funding source.

Then using Alby's NWC (Nostr Wallet Connect) function, connect to Zeus (using the NWC connection type) and done, you are using  your phoenixd server with Zeus as a regular LN wallet.

**Reply by mandelmonkey** [reply]
thanks that makes sense, I guess the advantage is its always on if you run pheonix d on a server and connect to it via mobile

**Reply by Darth-Coin** [reply]
Yes, I think was designed mainly to be more like a funding backend for Lnbits, btcpay, Alby hub, to start quickly a LN node and don't bother with liquidity management.
